app/controllers/posts_controller.rb

```diff
@@ -37,4 +37,9 @@ def show
     @user = User.find(params[:user_id])
```

Comment:
- [ ] Please ensure using `load_and_authorize_resource` as you make the best advantage of the CanCan gem. 🙏

Reply:
of course, thank you!